pkgs/development/libraries/qt-5/mkDerivation.nix

```diff
@@ -16,6 +16,9 @@ let
       optional (!debug) "-DQT_NO_DEBUG"
       ++ lib.toList (args.NIX_CFLAGS_COMPILE or []);
 
+    configureFlags = [ "-no-framework" ]
+                  ++ (args.configureFlags or []);
```

Comment:
@matthewbauer This broke `audaciousQt5` (https://hydra.nixos.org/build/74997027) and is out of place here because it is used for all packages defined with `libsForQt5.callPackage`. Could you move `-no-framework` away?

Reply:
Where can we put this though? I think qmake flags is the right place maybe?

Reply:
It seems that `-no-framework` should only apply to Qt configure, and should not be passed to any `qmake`. Am I right?

Reply:
Yes that will work. My confusion is from thinking that all qt derivations use the configure script. Most will just use qmake & hopefully -no-framework is remembered.

Reply:
Please try this place: https://github.com/NixOS/nixpkgs/blob/18.03/pkgs/development/libraries/qt-5/modules/qtbase.nix#L299

Reply:
Thank you for fixing this in dcea0913e5e66f44660b5de7451672bebee7b871!

/cc @vcunat Although this is a mass rebuild in `master`, and most packages were building with `-no-framework` because `configure` usually ignores unknown flags, so it might be better to revert this from `master`, merge `staging`, and then apply this to `staging`.

Reply:
Actually this fixed quite a few packages: https://hydra.nixos.org/eval/1460869#tabs-now-succeed

Reply:
@vcunat has cancelled them to free up resources for `staging` builds; there should be no manually cancelled builds once `staging` is merged.